packages/stockflux-launcher/src/App.js

```diff
@@ -1,66 +1,146 @@
-import React, { useRef, useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import cx from 'classnames';
 import { OpenfinApiHelpers } from 'stockflux-core';
-import { useDockWindow, ScreenEdge } from 'openfin-react-hooks';
+import {
+  useDocked,
+  useDockWindow,
+  useOptions,
+  ScreenEdge
+} from 'openfin-react-hooks';
 import {
   FaChevronUp,
   FaChevronLeft,
   FaChevronRight,
-  FaRegHandRock
+  FaRegHandRock,
+  FaUnlock
 } from 'react-icons/fa';
-import AppShortcuts from './app-shortcuts/AppShortcuts';
 import FreeTextSearch from './free-text-search/FreeTextSearch';
 import ToolBar from './toolbar/ToolBar';
 import CloseButton from './toolbar/CloseButton';
 import './App.css';
 
-const isLauncherHorizontal = edge => edge === ScreenEdge.TOP;
-
 export default () => {
+  const [isDockable, setIsDockable] = useState(false);
+  const [isDocked, setDocked] = useState(true);
```

Comment:
isDocked might be redundant because we already are storing what edge is the launcher docked to (line 30). If it;s not docked it's `ScreenEdge.None`.